### DISCUSSION TITLE
Support for exporting traces to CSV/JSON

### DISCUSSION THREAD

**Discussion by david1542** [langfuse/langfuse#3208] [Ideas]
### Describe the feature or potential improvement

Hi there,

We have an open source project that uses Langfuse to monitor our agent's interactions. Each instance of our open source would have its own Langfuse instance.

When debugging problems on a user instance, it's quite hard not seeing the traces. I was thinking it may be great to be able to "export" a specific trace. This way, they can send us the problematic interaction and we can debug it on our Langfuse instance.

**Comment by marcklingen** [comment]
Thanks for sharing. Is there a way to use public links to help debug problems?

https://langfuse.com/docs/tracing-features/url

**Comment by harrysondubs** [comment]
@marcklingen hey there 👋 My startup is currently using Langfuse to monitor our customers' use of our AI guest messaging system for Airbnb/VRBO hosts. We were also hoping to have the option to export traces (at least the meta-data columns such as date, tags, usage, cost, etc.), since we rely heavily on trace tags to understand how the LLM is interacting with guests. Right now I have to go in and manually copy-paste usage and tag data into CSV for it to provide any meaningful insights, so the ability to export the Trace table to CSV would be a huge help!

**Reply by harrysondubs** [reply]
Annnd I just saw that this is now a feature. Thank you!

Would also be cool to have some kind of Tags graphic chart in the dashboard

**Reply by marcklingen** [reply]
Yes, added by @marliessophie recently. Let us know if this doesn’t solve for your use case.

**Reply by marcklingen** [reply]
@harrysondubs I've added this thread to collaborate on tag-based dashboard requirements: https://github.com/orgs/langfuse/discussions/3476
